code/game/objects/structures/inflatable.dm

```diff
@@ -216,8 +216,8 @@
 	desc = "Contains inflatable walls and doors."
 	icon_state = "inf_box"
 	item_state = "syringe_kit"
-	max_combined_w_class = 21
 	w_class = WEIGHT_CLASS_NORMAL
+	can_hold = list(/obj/item/inflatable, /obj/item/inflatable/door)
```

Comment:
```suggestion
	can_hold = list(/obj/item/inflatable)
```
Should also work

Reply:
oh, it automatically does subtypes? neat

Reply:
Yep yep.
![image](https://user-images.githubusercontent.com/15887760/148683406-3fd914a5-eb92-4c8e-9f0d-c54199ed4ddd.png)
It will make a typecache of the items in the list. And by default `typecacheof` will make the entire tree of the items